src/main/java/seedu/address/model/Model.java

```diff
@@ -15,8 +16,13 @@ public interface Model {
     /** {@code Predicate} that always evaluate to true */
     Predicate<Person> PREDICATE_SHOW_ALL_PERSONS = unused -> true;
 
-    /** {@code Predicate} that always evaluate to true */
-    Predicate<InternshipApplication> PREDICATE_SHOW_ALL_APPLICATIONS = unused -> true;
+    /** {@code Predicate} that evaluate to true for all unarchived applications */
+    Predicate<InternshipApplication> PREDICATE_SHOW_ALL_APPLICATIONS = internshipApplication ->
+            internshipApplication.getStatus() != InternshipStatus.ARCHIVED;
```

Comment:
With this change, so the intended behaviour of `list` command no longer list out "all" applications right, I guess it may be more proper to name this as `PREDICATE_SHOW_ONGOING_APPLICATIONS`

Reply:
Good suggestion, will update affected parts accordingly